### PR TITLE
feature/main-menu

### PR DESCRIPTION
**Pull request by MichaelB-pl**
Some simple changes before specific code

### CODE REVIEW

File: src/styles/colors.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+$colorPrimary: #8bc34a;
+$colorPrimaryDark: #5a9216;
+$colorPrimaryLight: #bef67a;
+$colorPrimaryExtraLight:#f2ffab;
+$colorAccent:   #790e8b;
+
+
+$scrollbarTrackBackground: #0000;
```
Comment (sintari) [inline src/styles/colors.scss]:
```suggestion
$scrollbarTrackBackground: #000000;
```

Comment (sintari) [reply src/styles/colors.scss]: I would stick to convetion with 6 characters after `#`

File: src/styles/dimens.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+$phoneMaxHeight: 480px;
+
+$menuItemMaxWidth: 480px;
+
```
Comment (sintari) [inline src/styles/dimens.scss:3]: there is no need for 2 free lines
one is enough or even none

**Review by sintari — CHANGES_REQUESTED** [review]

File: src/styles/colors.scss (hunk 1 above)
Comment (MichaelB-pl) [reply src/styles/colors.scss]: But I need transparent color, not black.

**Review by sintari — APPROVED** [review]

**Pull request merged**